README.md

```diff
@@ -46,6 +46,19 @@ However this has a few surprising edge-cases:
  - UUIDs aren't caught by it. Again, `27375774-a49e-4fa6-9cb1-c2fdcc43b9d6` looks random, and we might even use it as a fairly secure password, but it's just lower-case hex. Hardly random.
  - You may even find some short truly random strings slip by because they are short and accidentally don't have enough complexity.
 
+## Uninstalling The Old Version
+Only one thing is needed to disable our old, customised, version of the pre-commit framework: Remove the global pre-commit hook from git:
+
```

Comment:
to avoid two colons in one sentence I might use:

`... the pre-commit framework – remove the global pre-commit hook from git:`

Reply:
Nice, thank you!